tests/functional/policies/check_triggers/e07.pass.te

```diff
@@ -1,3 +1,10 @@
-policy_module(w12, 1.0)
+policy_module(e07_pass, 1.0)
+
+allow source target:file read;
+
+allow source target:file { open read };
 
 allow source target:file read_file_perms;
+
+# TODO:
```

Comment:
What's the TODO for?

Reply:
The class `file` does not support the permission `search`.
But since this kind of check is not yet implemented, this statement still passes and is therefore in the `pass.te` file.
I'll improve the comment.